main.js

```diff
@@ -0,0 +1,32 @@
+
+var $car = document.querySelector('.car');
+
+var direction = 0;
+
+document.addEventListener('keydown', function(e) {
+
+  if(e.code === 'KeyA') {
+    direction--;
+
+    if(direction < 0) {
+      direction = 8 - (-1*direction);
+    }
+
+    $car.className = 'car direction' + direction;
+
+  } else if (e.code === 'KeyD') {
+    direction++;
+
+    if (direction > 7) {
+      direction = 0;
+    }
+
+    $car.className = 'car direction' + direction;
+
+  } else if (e.code === 'KeyS') {
+
+  } else if (e.code === 'KeyW') {
+
+  }
```

Comment:
This logic seems to be difficult and hard to read.  Less code is often not as good as readable code.

You can use words like up down left and right to dictate what direction the car should go.